libs/net/models/Areas/Services/Models/Content/ContentModel.cs

```diff
@@ -282,6 +282,16 @@ public ContentModel(Entities.Content entity) : base(entity)
             entity.Series = new Entities.Series(model.OtherSeries, model.SourceId);
         }
 
+        if (model.Product != null)
```

Comment:
Why have you added this?  It can result in the creation of new products when saving content.  Which is not desirable behaviour.

Reply:
Currently we have a product filter on the UI, without adding this the Content.Product will be null, which means we won't be able to use the product filter for the validation purpose.

Reply:
Updated with carrying over the existing product Id, thanks

Reply:
This model is used for adding and updating content.  Including child properties like this can lead to unexpected issues.  The filter should work with the primary key `ProductId`.

Reply:
Updated, thanks